README.md

```diff
@@ -1,250 +1,138 @@
-# are automatically mapped at this stage -- e.g., Department is mapped
-# because employee.department is a relationshp to Department)
-strawberry_sqlalchemy_mapper.finalize()
-# only needed if you have polymorphic types
-additional_types = list(strawberry_sqlalchemy_mapper.mapped_types.values())
-schema = strawberry.Schema(
-    query=Query,
-    mutation=Mutation,
-    extensions=extensions,
-    types=additional_types,
-)
-
-# You can now query, e.g.:
-"""
-query {
-    departments {
-        id
-        name
-        employees {
-            edge {
-                node {
-                    id
-                    name
-                    department {
-                        # Just an example of nested relationships
-                        id
-                        name
-                    }
-                }
-            }
-        }
-    }
-}
-"""
-```
+    def departments(self) -> List[DepartmentType]:
+        return session.scalars(select(Department)).all()
 
-## Limitations
-
-SQLAlchemy Models -> Strawberry Types and Interfaces are expected to have a consistent
-(customizable) naming convention. These can be configured by passing `model_to_type_name`
-and `model_to_interface_name` when constructing the mapper.
-
-Natively supports the following SQLAlchemy types:
-
-```python
-Integer: int,
-Float: float,
-BigInteger: BigInt,
-Numeric: Decimal,
-DateTime: datetime,
-Date: date,
-Time: time,
-String: str,
-Text: str,
-Boolean: bool,
-Unicode: str,
-UnicodeText: str,
-SmallInteger: int,
-SQLAlchemyUUID: uuid.UUID,
-VARCHAR: str,
-ARRAY[T]: List[T] # PostgreSQL array
-JSON: JSON # SQLAlchemy JSON
-Enum: (the Python enum it is mapped to, which should be @strawberry.enum-decorated)
+schema = strawberry.Schema(query=Query)
+mapper.finalize()  # Finalize all mapped types
 ```
 
-Additional types can be supported by passing `extra_sqlalchemy_type_to_strawberry_type_map`,
-although support for `TypeDecorator` types is untested.
-
-Association proxies are expected to be of the form `association_proxy('relationship1', 'relationship2')`,
-i.e., both properties are expected to be relationships.
-
-Roots of polymorphic hierarchies **are supported**, but are also expected to be registered via
-`strawberry_sqlalchemy_mapper.interface()`, and its concrete type and
-its descendants are expected to inherit from the interface:
-
-```python
-class Book(Model):
-    id = Column(UUID, primary_key=True)
-
-
-class Novel(Book):
-    pass
-
-
-class ShortStory(Book):
-    pass
-
-
-# in another file
-strawberry_sqlalchemy_mapper = StrawberrySQLAlchemyMapper()
-
-
-@strawberry_sqlalchemy_mapper.interface(models.Book)
-class BookInterface:
-    pass
-
-
-@strawberry_sqlalchemy_mapper.type(models.Book)
-class Book:
-    pass
-
-
-@strawberry_sqlalchemy_mapper.type(models.Novel)
-class Novel:
-    pass
-
-
-@strawberry_sqlalchemy_mapper.type(models.ShortStory)
-class ShortStory:
-    pass
-```
+## Feature Status
```

Comment:
I still think we should bring this feature status to the start of the readme, before the Getting Started.
I also think its better to we just keep the old text, since it has some info we need, what you think?